packages/react/src/components/InAppMessaging/MessageLayout/MessageLayout.tsx

```diff
@@ -118,7 +125,7 @@ export function MessageLayout({
               onClick={secondaryButton?.onAction}
               style={styles.secondaryButton}
             >
-              {secondaryButton?.title}
+              {secondaryButton.title}
```

Comment:
The optional chaining here is unnecessary since we know that `secondaryButton` is defined from line 119 (same thing with line 140)